spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "rails_helper"
+require './app/helpers/sessions_helper'
+
+RSpec.configure do |c|
+  c.include SessionsHelper
+end
+RSpec.describe UsersController, type: :controller do
+  let(:user) {FactoryBot.create :user}
```

Comment:
`let` và `let!` có giống nhau ko em

Reply:
Khác nhau chị:
+ let : Tạo ra khi nào gọi nó.
+ let!: Được gọi tự động cho mỗi it